### PR TITLE
RFR - Menu and Show Page Stylin' (plus some other fixes)

### PR DESCRIPTION
**Pull request by KellyTeresa**

### CODE REVIEW

File: app/assets/stylesheets/_variables.scss [hunk 1]
```diff
@@ -0,0 +1,23 @@
+$white          : #FFFFFF;
+$ghost          : #FAFAFA;
+$snow           : #F9F9F9;
+$vapor          : #F6F6F6;
+$white-smoke    : #F5F5F5;
+$silver         : #EFEFEF;
+$smoke          : #EEEEEE;
+$gainsboro      : #DDDDDD;
+$iron           : #CCCCCC;
+$base           : #AAAAAA;
```
Comment (KellyTeresa) [inline app/assets/stylesheets/_variables.scss]: really, hound? you really care that much???

File: app/assets/stylesheets/_variables.scss [hunk 2]
```diff
@@ -0,0 +1,23 @@
+$white          : #fff;
+$ghost          : #fAfAfA;
```
Comment (KellyTeresa) [inline app/assets/stylesheets/_variables.scss]: fafafa at that hound dog. both are accepppttaaabbleeee

File: app/helpers/session_helper.rb [hunk 3]
```diff
@@ -2,4 +2,12 @@ module SessionHelper
...
     user && (user.admin? || user == item.user)
   end
+
+  def display_user_avatar(item)
+    if item.user.profile_photo.url
+      image_tag(item.user.profile_photo.url)
+    else
+      image_tag("default-dollar-drainer.png")
+    end
+  end
```
Comment (davidrf) [inline app/helpers/session_helper.rb]: Check out [this section](https://github.com/carrierwaveuploader/carrierwave#providing-a-default-url) of the carrierwave documentation. You can set a default image if the user did not upload one. You just specify the directory where you want the image to be in your app, and then you can just do `user.profile_photo`

File: app/views/plans/show.html.erb [hunk 4]
```diff
@@ -1,31 +1,38 @@
...
-        <h3>Modify Plan</h3>
-        <p><%= link_to 'Edit Plan', edit_plan_path(@plan) %></p>
-        <p><%= link_to 'Delete Plan', plan_path(@plan), method: :delete,
-        data: { confirm: "Really delete this plan?" } %></p>
-      <%- end -%>
+  <div class="small-12 medium-5 columns">
+    <div class="row collapse">
+      <div class="plan-creator small-6 medium-7 columns">
+        <h5>Plan created by:</h5>
+        <%= display_user_avatar(@plan) %>
```
Comment (davidrf) [inline app/views/plans/show.html.erb]: This can now be `@plan.user.profile_photo.url`

File: app/views/plans/show.html.erb [hunk 5]
```diff
@@ -36,6 +43,8 @@
           <li>
             <%="#{review.rating} stars."%>
             <%=" #{review.comment}" unless review.comment.nil? %>
+            <%= display_user_avatar(review) %>
```
Comment (davidrf) [inline app/views/plans/show.html.erb]: This can now be <%= image_tag(@review.user.profile_photo.url) %>

**Pull request merged**